plugin-gradle/README.md

```diff
@@ -527,6 +528,21 @@ spotless {
 
 For details, see the [npm detection](#npm-detection) and [`.npmrc` detection](#npmrc-detection) sections of prettier, which apply also to tsfmt.
 
+## JSON
+
+- `com.diffplug.gradle.spotless.JsonExtension` [javadoc](https://javadoc.io/static/com.diffplug.spotless/spotless-plugin-gradle/5.12.4/com/diffplug/gradle/spotless/JsonExtension.html), [code](https://github.com/diffplug/spotless/blob/main/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JsonExtension.java)
+
+```gradle
+spotless {
+  json {
+    target '**/*.json' // you have to set the target manually
+
+    indentWithSpaces(6)
+    indentWithTabs()
```

Comment:
If we keep `JsonExtension` (see comment there), users will definitely still be able to swap for different json formatters. e.g.

```gradle
spotless {
  json {
    target 'src/**/*.json' // have to set the target manually
    prettier()   // requires npm, slow to setup
    eclipseWtp() // somewhat complex
    simple()     // simple
```